File: qualifier-week-3/graded assignments/GrPA5 - Graded.c
#include <stdio.h>
#include <math.h>
int main() {
    double a, b, c;
    scanf("%lf %lf %lf", &a, &b, &c);
    // Write solution code below
    double s, area;
    s = (a + b + c) / 2;
    area = sqrt(s * (s - a) * (s - b) * (s - c));
    printf("%.2lf", area);
    return 0;
}

File: qualifier-week-3/graded assignments/GrPA5 - Graded.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var a, b, c, s float64
	fmt.Scan(&a, &b, &c)
	s = (a + b + c) / 2
	println(math.Sqrt(s * (s - a) * (s - b) * (s - c)))
}
